main/cmds.go

```diff
@@ -72,44 +72,40 @@ func install(ctx *log.Context, h HandlerEnvironment, seqNum int) (string, error)
 	// If it is (2) attempt to migrate to mrseq.  Find the latest settings
 	// file, and set the sequence to it.
 	if _, err := os.Stat(mostRecentSequence); os.IsNotExist(err) {
-		migrateToMostRecentSequence(ctx, h, seqNum)
+		err := migrateToMostRecentSequence(ctx, seqNum)
```

Comment:
Previously, install would never fail for this extension. Given how often this extension is used, I recommend keeping the current behavior since this change is only for logging.